magefiles/mage.go

```diff
@@ -367,7 +367,6 @@ func (Test) CgoDocker(ctx context.Context) error {
 	env["ARCH"] = build.Default.GOARCH
 	env["CI_PROJECT_DIR"] = "/opt"
 	env["ENVIRONMENT"] = "dev"
-	env["GOPATH"] = build.Default.GOPATH
```

Comment:
CGO tests wouldn't work locally. This somewhat helped because tests run now, but there is also permission issue and tests that are manipulating iptables for example are failing now... Decided not to bother with that now.